README.md

```diff
@@ -184,6 +184,11 @@ A curated list of awesome things related to <a href='https://github.com/vitejs/v
 
 - Check the [Awesome Rollup list](https://github.com/rollup/awesome) for community maintained rollup plugins, and refer to the Vite Docs section about [rollup plugin compatibility](https://vitejs.dev/guide/api-plugin.html#rollup-plugin-compatiblity).
 
+## Integrations with Backends
+
```

Comment:
```suggestion

### Ruby on Rails

```

Would think this could be better? I am not familiar with ruby / rails, is the subtitle correct?

Reply:
Yeah, was thinking of keeping it open in case people add support in other languages/frameworks, but we can use a more specific title 👍